GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+# Governance
+
+## Overview
+
+sktime is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project, and participate in the governance process. This document describes how that participation takes place and how to set about earning merit within our community. 
+
+We are particularly motivated to support new and/or anxious collaborators, people who are looking to learn and develop their skills, and anyone who has experienced discrimination in the past.
```

Comment:
while strictly speaking correct, the reference to discrimination is a bit unusual, since it reference negative experiences - so it carries a strange sub-vibe that we like community members to have had past negative experiences. Maybe "diversity" type phrasing is less awkward here.

Reply:
It's the same in the CoC, but happy to replace it with other formulations, any ideas?